docs/en/query_language/functions/encoding_functions.md

```diff
@@ -1,5 +1,8 @@
 # Encoding functions
 
+## char
+Accepts multiple arguments of `Number` types. Returns a string, each char of the results corresponds to the ascii character of the input numbers. Same behavior as [MySQL char function](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_char);
```

Comment:
Does it really have the same behaviour? What about numbers out of ASCII range?

Reply:
I gave up to implement the same Behavior as mysql when number out of  ASCII range.
It could return multiply characters, I think it's unnecessary and I don't know the internal implementation.
```
mysql> select char(2221111111111);
+---------------------+
| char(2221111111111) |
+---------------------+
| $���                   |
+---------------------+
1 row in set (0.00 sec)
```

 So I just took the first byte. It's easy to make it done by Column and easy to know the results.